recipe/meta.yaml

```diff
@@ -24,13 +25,16 @@ requirements:
     - setuptools
     - wheel
   run:
-    - python >=3.6.2
-    - astroid >=2.9.0,<2.10
+    - python
+    - astroid >=2.11.6,<2.12.0
     - colorama  # [win]
+    - dill >=0.2
     - isort >=4.2.5,<6
     - mccabe >=0.6,<0.8
     - platformdirs >=2.2.0
-    - toml >=0.9.2
+    # 2022/07/25 RSK pylint switched from toml to tomli in 2.13.0
+    - tomli >=1.1.0
```

Comment:
`tomli >=1.1.0` requires only for `py<311` https://github.com/PyCQA/pylint/blob/566ffca72a1a08c9e7ec14383c4710d64165c592/setup.cfg#L53